README.md

```diff
@@ -131,6 +131,9 @@ kube-system                kubernetes-dashboard-5f7b999d65-ntrxb         1/1
 ```
 
 2. Create a cluster:
+
+You should review files `cluster.yaml`, `machines.yaml`, and ensure the domain and dataCenter and osReferenceCode are set. If you want to do any customization now is the time to do it.
```

Comment:
s/domain/`domain`
s/dataCenter/`dataCenter`
s/osReferenceCode/`osReferenceCode`

You should review files `cluster.yaml` and `machines.yaml`, ensure the `domain`, `dataCenter` and `osReferenceCode` are set. You can customize those values based on your requirement.

Reply:
Thanks @gyliu513's comment and Addressed.